app/views/messages/index.html.erb

```diff
@@ -19,9 +19,11 @@
 <section class="message--create">
   <div class="grid">
     <div class="grid__item width-two-thirds shift-one-sixth">
-      <%= form_for @message, remote: true, :html => {:class => "sendbar form-width--searchbar"}, builder: GcfFormBuilder do |f| %>
-      <%= f.gcf_send_message_field :body, "Send", autofocus: true, placeholder: "Send a text message" %>
-      <input name="client_id" type="hidden" value="<%= @client.id %>">
+      <%= form_for @message, remote: true, :html => {class: 'sendbar form-width--searchbar' }, builder: GcfFormBuilder do |f| %>
```

Comment:
Regarding `builder: GcfFormBuilder`, you can set this globally:

https://github.com/multnomah-fellowship/myadvocate/commit/89ddcde2453b33d6e1c0299a41f979db745b9778#diff-9df40b149f57e1efda80613f77066095